.eslintrc

```diff
@@ -6,12 +6,12 @@
     "mocha": true
   },
   "rules": {
-    "semi": 2,
+    "semi": [2, "always"],
```

Comment:
According to http://eslint.org/docs/rules/semi#options , "always" is the default, so I think this change doesn't do anything.